recipe/meta.yaml

```
@@ -36,6 +36,7 @@ outputs:
     requirements:
       build:
       host:
+        - sysroot_{{ target_platform }}
       run:
```

We need to add the following, merge this, merge the compiler and then remove this.
```suggestion
      run:
        - sysroot_{{ target_platform }}
```

---

Ahhhhhh. That's a fun game!

---

I see. We need to boostrap the whole thing with the sysroot around. Makes sense.